### PR TITLE
Added NOTICE.txt

### PR DESCRIPTION
**Pull request by hmhealey**

### CODE REVIEW

**Comment by enahum** [comment]
@esethna once you review you can merge it

File: NOTICE.txt [hunk 1]
```diff
@@ -0,0 +1,567 @@
...
+SOFTWARE.
+
+---
+
+## keymirror
+
+This product contains a modified portion of 'keymirror', to create an object with values equal to its key names.
+
+* HOMEPAGE: 
+  * https://github.com/STRML/keyMirror
```
Comment (esethna) [inline NOTICE.txt]: This page does not have a license in the repo. Are we sure we can use this?

File: NOTICE.txt [hunk 2]
```diff
@@ -0,0 +1,567 @@
...
+  
+* LICENSE: 
+
+Apache License 2.0
```
Comment (esethna) [inline NOTICE.txt]: How do you know this is an Apache License?

File: NOTICE.txt [hunk 3]
```diff
@@ -0,0 +1,567 @@
...
+
+Copyright (c)
```
Comment (esethna) [inline NOTICE.txt]: Needs a name here

File: NOTICE.txt [hunk 4]
```diff
@@ -0,0 +1,567 @@
...
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+---
+
+## react-addons-pure-render-mixin
+
+This product contains a modified portion of 'react-addons-pure-render-mixin',  a JavaScript library for building user interfaces.
+
+* HOMEPAGE: 
+  * https://github.com/facebook/react
```
Comment (esethna) [inline NOTICE.txt]: Same page as above. Not sure why these are two separate listings? Is there a page specific to "react-addons-pure-render-mixin"?

**Comment by esethna** [comment]
@hmhealey Reviewed. I added a commit for minor tweaks but I wasn't sure about the above comments. Please review and circle back?

File: NOTICE.txt [hunk 5]
```diff
@@ -0,0 +1,566 @@
...
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+---
+
+## react-addons-pure-render-mixin
+
+This product contains a modified portion of 'react-addons-pure-render-mixin',  a JavaScript library for building user interfaces by Facebook, Inc.
+
+* HOMEPAGE: 
+  * https://github.com/facebook/react
```
Comment (esethna) [inline NOTICE.txt]: Same link as above. Is there a page specifically for "react-addons-pure-render-mixin"?

File: NOTICE.txt (hunk 4 above)
Comment (hmhealey) [inline NOTICE.txt]: This is also copied from the platform repo. You're right though; I don't think we need to include subpackages in here

**Comment by hmhealey** [comment]
@esethna Updated the PR to match the changes I made to the platform repo (copying keymirror into utils and removing the attribution of react-addons-pure-render-mixin from NOTICE.txt)

**Comment by enahum** [comment]
@hmhealey needs rebase

**Pull request merged**